decidim-surveys/app/forms/decidim/surveys/survey_answer_form.rb

```diff
@@ -24,6 +26,10 @@ def map_model(model)
       def survey
         @survey ||= Survey.where(feature: current_feature).first
       end
+
+      def body_not_blank
+        errors.add("body", :blank) if body.all?(&:blank?)
```

Comment:
Maybe I'm missing something, why is body an array? What I meant is that if any element of the array is blank we should fail.